lib/Doctrine/DBAL/Connection.php

```diff
@@ -1093,7 +1093,7 @@ public function lastInsertId($seqName = null)
      *
      * @param \Closure $func The function to execute transactionally.
      *
-     * @return void
+     * @return mixed
```

Comment:
Maybe a little description about what is returned here might make sense so that the API user knows it returns the result from the callback